packages/core/core/src/Config.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+import nullthrows from 'nullthrows';
+import path from 'path';
+
+import type {FilePath, PackageName} from '@parcel/types';
+
+type ConfigOpts = {|
+  searchPath: FilePath,
+  resolvedPath?: FilePath,
+  result?: any,
```

Comment:
can we avoid this `any`?

Reply:
Offline discussion for posterity:

Seems like we would have to add a Generic to Transformer and return `{result, devDeps, includedFiles, watchGlob}` and what not instead of passing in a config object that gets mutated. Something to explore, but not a blocker for this PR since `getConfig` currently returns `any` so there's no regression.